plugins/imagebase/plugin.js

```diff
@@ -535,14 +538,14 @@
 		 */
 		return {
 			setUp: function( editor ) {
-				var listeners = [];
+				var listeners = [],
+					// Changing the scope of previous so it actually stores previously focused element (#1543).
```

Comment:
This comment is understandable only in relation to previous code. As this code is not super complex I would skip this comment here.